src/components/Room/Room.tsx

```diff
@@ -61,6 +65,14 @@ export function Room({
     await sendMessage(message)
   }
 
+  const showMessages = roomContextValue.isShowingMessages
+
+  const { showRoomControls } = useContext(ShellContext)
+
+  const [display, setDisplay] = useState(Dimensions.get('window'))
+  Dimensions.addEventListener('change', ({ screen }) => setDisplay(screen))
```

Comment:
I'm not really sure what the cheapest implementation is to listen to window size changes, this is just something I found in a google search. any better suggestions?

Reply:
There are a variety of minimal ways to achieve this. It would be ideal to use an external implementation that we could pull in as an NPM dependency. I found this package that seems like it might be worth giving a try: https://www.npmjs.com/package/@react-hook/window-size. According to [Bundlephobia](https://bundlephobia.com/package/@react-hook/window-size@3.1.1) it's only 1.6kB, so it would not impact our bundle size significantly.

If that doesn't work for whatever reason, this implementation seems like a pretty copy/paste-able solution: https://usehooks.com/useWindowSize/